.github/renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "local>cognitedata/renovate-config"
+    ],
+    "git-submodules": {
+        "enabled": true
+    },
+    "labels": [
+        "dependencies",
+        "auto-merge"
+    ]
+}
```

Comment:
Missing newline at the end of this file

Reply:
I'm not sure that really matters? I can add it and push, but its JSON and that line can never change...